codalab_service.py

```diff
@@ -249,6 +249,16 @@ def has_callable_default(self):
     # User
     CodalabArg(name='user_disk_quota', help='How much space a user can use', default='100g'),
```

Comment:
Same question as above: shouldn't this default be changed so that the time and disk quota for non-edu users is less?

Reply:
For our deployment, the actual disk / time quota is controlled by environmental variables `CODALAB_USER_DISK_QUOTA` and `CODALAB_USER_TIME_QUOTA`. But for other people who want to deploy their own Codalab instances, they might want edu users and normal users have same quota.